js/scripts/util.js

```diff
@@ -176,12 +176,21 @@ const checkBlockCollisions = (entity) => {
 /** Contains all functions called as event handlers. */
 const EVENT_HANDLERS = {
     gameplayMouseDown: (e) => {
-        GAME.user.firing = false;
-        GAME.user.aiming = true;
+        // check if mouse button is left click
```

Comment:
It's a little confusing to me that the comment says you're checking for a left click, but the first condition is checking for right click. Not a big deal at all though.